src/docs/deployment/web.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Preparing a web app for release
+description: How to prepare for and release a web app.
+short-title: Web
+---
+
+During a typical development cycle,
+you test an app using `flutter run -d chrome`
+(for example) at the command line.
+This builds a _profile_ version of your app.
+
+This page helps you prepare a _release_ version
+of your app.
+
+This page covers the following topics:
+
+* [Adding a launcher icon](#adding-a-launcher-icon)
+* [Signing the app ??](#signing-the-app)
+* [Building the app for release](#building-the-app-for-release)
+* [Deploying to the web](#deploying-to-the-web)
+
+## Adding a launcher icon
+
+TBD
+
+## Signing the app
+
+Is signing an available option for web apps?
+
+What about obfuscation and minification?
+Are web apps automatically obfuscated and minified?
+
+## Building the app for release
+
+Build the app for deployment using the
+`flutter build web` command.
```

Comment:
by default this builds a release version, it can be configured to build the profile version specifically. There is no artifact for a debug build, because it relies on thousands of tiny little files and very specifically configured web server